lib/services/project-service.ts

```diff
@@ -50,10 +51,12 @@ export class ProjectService implements IProjectService {
 
 			await this.ensureAppResourcesExist(projectDir);
 
-			let packageName = constants.TNS_CORE_MODULES_NAME;
-			await this.$npm.install(packageName, projectDir, { save: true, "save-exact": true });
-
 			let templatePackageJsonData = this.getDataFromJson(templatePath);
+
+			if (!(templatePackageJsonData && templatePackageJsonData.dependencies && templatePackageJsonData.dependencies[constants.TNS_CORE_MODULES_NAME])) {
```

Comment:
the `tns-core-modules-widgets` are dependency of the `tns-core-modules`. In case they exist in the template, it means someone has a very good reason to add them. I suggest to skip checks for them until we find a proper use-case.